spec/acceptance/nodesets/default.yml

```diff
@@ -0,0 +1,18 @@
+HOSTS:
```

Comment:
please use the env variables shown in https://github.com/simp/pupmod-simp-hirs_provisioner/blob/master/spec/acceptance/nodesets/default.yml